detoks/src/main/java/nl/tudelft/trustchain/detoks/Profile.kt

```diff
@@ -41,6 +50,16 @@ class Profile(
         profiles[key]!!.hopCount = hopCount
     }
 
+    fun updateEntryLikes(key: String, likes: Int, myUpdate: Boolean) {
+        addProfile(key)
+        if(myUpdate) {
+            profiles[key]!!.likes += (likes / NetworkSizeGossiper.networkSizeEstimate)
```

Comment:
```suggestion
            profiles[key]!!.likes += (1 / NetworkSizeGossiper.networkSizeEstimate)
```

Since you can only add one like when it is your own update